openapi.yaml

```diff
@@ -3841,6 +3868,13 @@ components:
               title: Thumbnail
               roles:
                 - thumbnail
+            process.json:
+              href: 'https://openeo.org/api/download/583fba8b2ce583fba8b2ce/process.json'
```

Comment:
just a general thought: maybe use "fictional" domain like  `example.openeo.org` instead of a real domain in all examples?

Reply:
Good idea...